src/app/services/ObservedPartService.ts

```diff
@@ -118,6 +124,18 @@ export class ObservedPartService implements OnDestroy {
             }
         }
     }
+    public getObservedPart(): Promise<MGPOptional<ObservedPart>> {
+        // Need to make sure we fully initialized, hence observedPartObs contains a value
```

Comment:
We need to make sure that we are fully...

Reply:
changé en we *have* pour emphaser le fait que c'est déjà fait